src/magicbox/ResultPreviewsManager.ts

```diff
@@ -0,0 +1,212 @@
+import { $$, Dom } from '../utils/Dom';
+import { l } from '../strings/Strings';
+import { defaults, findIndex } from 'lodash';
+import { Component } from '../ui/Base/Component';
+import { Direction } from './SuggestionsManager';
+
+export interface IPopulateSearchResultPreviewsEventArgs {
```

Comment:
`IPopulateSearchResultPreviewsEventArgs` and `ResultPreviewsManagerEvents` should be moved to `src/events`

Reply:
Alright, I'll move them.